BO4E/ENUM/BDEWArtikelnummer.cs

```diff
@@ -118,9 +118,15 @@ public enum BDEWArtikelnummer
         ENTGELT_EINBAU_BETRIEB_WARTUNG_MESSTECHNIK,
 
         /// <summary>9990001000631</summary>
+        /// <remarks>Ausgleichsenergie Überdeckung</remarks>
         [ProtoEnum(Name = nameof(BDEWArtikelnummer) + "_" + nameof(AUSGLEICHSENERGIE))]
         AUSGLEICHSENERGIE,
 
+        /// <summary>9990001000805</summary>
```

Comment:
was sagen einem die summaries hier?

Reply:
Das ist die tatsächliche in der Edifact zu findende Artikelnummer und ich glaube aus der Summary wird dann irgendwie der Name gelesen und zu dem tatsächlichen enum.